lib/modules/manager/bazel/parser.ts

```diff
@@ -0,0 +1,201 @@
+import crypto from 'crypto';
+import is from '@sindresorhus/is';
+import { lang, query as q } from 'good-enough-parser';
```

Comment:
@Churro If you're interested to take a look here is another approach to language parsing being used

Reply:
Thanks for the hint! I'll play with it next week :)